lib/node_modules/@stdlib/random/base/mt19937/lib/factory.js

```diff
@@ -153,7 +197,7 @@ function factory( seed ) {
 	* // returns <number>
 	*/
 	function normalized() {
-		return ( mt19937() - 1 ) * ( 1.0 / 4294967295.0 );
+		return ((( mt19937() - 1.0 )) >>> 0) / (NORMALIZATION_CONSTANT >>> 0);
```

Comment:
What is the rationale for ensuring `uint32` operands, here?

Reply:
Ah...another thing. The normalized algo should be the algo for generating 53 bits of randomness, not just 32 bits of randomness. That corresponds to function `genrand_res53` in the source. The approach is slightly deficient, as documented in a paper linked to in the RFC, but fine for most purposes.